src/PackageCompiler.jl

```diff
@@ -748,6 +750,8 @@ function create_app(package_dir::String,
                     sysimage_build_args::Cmd=``,
                     include_transitive_dependencies::Bool=true)
     warn_official()
+    # We call this at the very beginning to make sure that the user has a compiler available. Therefore, if no compiler is fiund, we throw an error immediately, instead of making the user wait a while before the error is thrown.
```

Comment:
```suggestion
    # We call this at the very beginning to make sure that the user has a compiler available.  Therefore, if no compiler 
    # is found, we throw an error immediately, instead of making the user wait a while before the error is thrown.
```